### PR TITLE
Add release GitHub Action for golang app

### PR DESCRIPTION
**Pull request by mbtamuli**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+
```
Comment (mugdha-adhav) [inline .github/workflows/release.yml]: Nit: Please remove this extra line.

File: .goreleaser.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    goos:
+      - linux
+      - darwin
+    goarch:
+      - amd64
+    binary: go-webapp
+    dir: minimal-go-app
+archives:
+  - format: binary
+    name_template: "{{ .Binary }}_{{ .Os }}_{{ .Arch }}"
```
Comment (mugdha-adhav) [inline .goreleaser.yml:16]: `{{ .Binary }}_{{ .Os }}_{{ .Arch }}`
Are these predefined variables? I would ideally like `.Os` to be `.OS`

**Review by mugdha-adhav — APPROVED** [review]
I haven't worked with goreleaser before. So please consider this review to be coming from a begineer.

File: .goreleaser.yml (hunk 2 above)
Comment (mbtamuli) [reply .goreleaser.yml:16]: These are variable from GoReleaser. Can't change.

**Pull request merged**